tests/test_benchmark.py

```diff
@@ -998,14 +999,14 @@ def lib_builder(name, native, env_init):
   def test_zzz_bullet(self):
     self.emcc_args.remove('-Werror')
     self.emcc_args += ['-Wno-c++11-narrowing', '-Wno-deprecated-register', '-Wno-writable-strings']
-    src = open(test_file('third_party', 'bullet', 'Demos', 'Benchmarks', 'BenchmarkDemo.cpp'), 'r').read()
-    src += open(test_file('third_party', 'bullet', 'Demos', 'Benchmarks', 'main.cpp'), 'r').read()
+    src = read_file(test_file('third_party/bullet/Demos/Benchmarks/BenchmarkDemo.cpp'))
+    src += read_file(test_file('third_party/bullet/Demos/Benchmarks/main.cpp'))
 
     def lib_builder(name, native, env_init):
-      return self.get_library(os.path.join('third_party', 'bullet'),
-                              [os.path.join('src', '.libs', 'libBulletDynamics.a'),
-                               os.path.join('src', '.libs', 'libBulletCollision.a'),
-                               os.path.join('src', '.libs', 'libLinearMath.a')],
+      return self.get_library(str(Path('third_party/bullet')),
```

Comment:
Others below are just `Path(..)`... Why is this `str(Path(..))`?

Reply:
Because get_library currently still expects a string for arg0.   We could change that as a followup, this is kind of an incremental step.